### ISSUE TITLE
crop image?

### ISSUE THREAD

**Issue by nbao** [ailon/markerjs2#163]
Hello,
I would like to ask if there is a function to crop the image before saving? If yes how can i do it?

**Comment by ailon** [comment]
No. There's a separate [CROPRO](https://github.com/ailon/cropro) library but there's no cropping built-in into marker.js.

**Comment by nbao** [comment]
![image](https://github.com/ailon/markerjs2/assets/5112514/567cbcd7-2631-4fb0-88e5-24f85f354ec9)
I got 2 error after  show markerArea

```
function showMarkerArea(target) {
              const markerArea = new markerjs2.MarkerArea(target);
              markerArea.addEventListener("render", (event) => {
                target.src = event.dataUrl;
              });
              markerArea.show();
              markerArea.addEventListener("close", (event) => {
                const base64Image = target.src;
                const filename = "screenshot.png";
                saveImageToFile(base64Image, filename);
              });
            }
```

**Comment by ailon** [comment]
This could happen if you are creating MarkerArea before your target image loaded or if it's within some container with, say, `position: relative`. In that case make sure you set [`targetRoot`](https://markerjs.com/reference/classes/markerarea#targetroot) appropriately.

P.S.: please don't reply with totally unrelated questions in the same issue. I'm closing this one for now. Please start a new issue if you for each specific problem/quesiton. Thanks!
